src/signals_notebook/entities/users/user.py

```diff
@@ -0,0 +1,160 @@
+
+    class Config:
+        validate_assignment = True
+        allow_population_by_field_name = True
+
+    @classmethod
+    def _get_endpoint(cls) -> str:
+        return 'users'
+
+    @classmethod
+    def create(cls, request: UserCreationBody) -> 'User':
+        """Create new User
+
+        Returns:
+            User
+        """
+        api = SignalsNotebookApi.get_default_api()
+        log.debug('Create User: %s...', cls.__name__)
+
+        response = api.call(
+            method='POST',
+            path=(cls._get_endpoint(),),
+            json={
+                'data': {
+                    'attributes': request.dict(by_alias=True, exclude_none=True),
+                }
+            },
+        )
+        log.debug('User: %s was created.', cls.__name__)
+
+        result = UserResponse(**response.json())
+        return cast(ResponseData, result.data).body
+
+    def refresh(self) -> None:
+        """Refresh user with new changes values
+
+        Returns:
+
+        """
+        from signals_notebook.entities import UserStore
+
+        UserStore.refresh(self)
+
+    def save(self) -> None:
+        """Update attributes of a specified user.
+
+        Returns:
+
+        """
+        api = SignalsNotebookApi.get_default_api()
+        api.call(
+            method='PATCH',
+            path=(
+                self._get_endpoint(),
+                self.id,
+            ),
+            json={
+                'data': {
+                    'attributes': self.dict(
+                        by_alias=True,
+                        include={
+                            'alias',
+                            'country',
+                            'first_name',
+                            'last_name',
+                            'organization',
+                        },
+                    )
+                },
+            },
+        )
+
+    def delete(self):
+        """Make specified user disabled.
+
+        Returns:
+
+        """
+        api = SignalsNotebookApi.get_default_api()
+        log.debug('Disable User: %s...', self.id)
+
+        api.call(
+            method='DELETE',
+            path=(self._get_endpoint(), self.id),
+        )
+        log.debug('User: %s was disabled successfully', self.id)
+
+    @property
+    def picture(self) -> Optional[File]:
+        if self._picture:
+            return self._picture
+
+        api = SignalsNotebookApi.get_default_api()
+        response = api.call(
+            method='GET',
+            path=(self._get_endpoint(), self.id, 'picture'),
+        )
+        if response.content == b'':
+            return self._picture
+
+        content_disposition = response.headers.get('content-disposition', '')
+        content_type = response.headers.get('content-type', '')
+        _, params = cgi.parse_header(content_disposition)
+        file_name = f'{self.first_name}_{self.last_name}.{content_type.split("/")[-1]}'
```

Comment:
Have u got any idea how to create file_name?
I wrote this and understood it was not optimal solution.
I dont prefer 'split' method and so can you solve this by your way?